app/views/game_masters/new.html.erb

```diff
@@ -5,9 +5,25 @@
   </h2>
   <h3><%= @table.long_name %></h3>
 </div>
-<h4>Add GM to Table</h4>
+
+
+<div class="well">
+  <div class="field">
+    Description: <%= @table.scenario.description %>
+  </div>
+  <% if @table.scenario.author.present? %>
+    <div>by <%= @table.scenario.author %></div>
+  <% end %>
+  <%= render partial: 'events/table_badges', locals: {table: @table} %>
+</div>
 <%= render 'form' %>
 
+<div class="alert-info">Once you have volunteered to GM a table, you will need to contact the hosts of the event (either
```

Comment:
Contact the hosts of the event because why?

Reply:
Because we don't want folks dropping their GM slots willy-nilly. 
We want them to commit when they select their GM slots, and then talk to us to drop them, so that _we know that they were dropped_.

This is on the theory to make it hard to un-volunteer. 

AJ and I talked a lot about this.

Reply:
Yes, but I don't think there's any part of that sentence that says '... to remove your volunteer position'. Unless it went off the edge and I missed it.

Reply:
that's the next line... 

```
<div class="alert-info">Once you have volunteered to GM a table, you will need to contact the hosts of the event (either
  by replying to your confirmation email, or through <a href="email:<%= ENV['GMAIL_SMTP_USERNAME'] %>?Subject=I%20can%20no%20longer%20GM%20something">our contact email</a>
```

Reply:
Hope that makes more sense!
_(note, sometimes the diffs are a little too tight to really tell what's going on!)_

Reply:
I'm still not seeing the next line. @_@ The text without code bits in it's entirety is "Once you have volunteered to GM a table, you will need to contact the hosts of the event (either by replying to your confirmation email, or through our contact email" ? If so, wouldn't it need an end to say something like "... if you wish to change your GM assignment" ? Or is there some other context on the page for removing things?